webapp/test_results_handler.go

```diff
@@ -12,6 +12,8 @@ import (
 	"net/http"
 	"strings"
 
+	"github.com/deckarep/golang-set"
```

Comment:
Nit: can we explicitly alias this to `mapset` like we always do? I think it's a good practice because otherwise it's not obvious what the package name is.